cogs/__init__.py

```diff
@@ -0,0 +1,7 @@
+import os
+from os.path import abspath, join, dirname, exists
+
+BASE_PATH = dirname(dirname(abspath(__file__)))
```

Comment:
Why was this stuff moved here? Especially the BASE_PATH? I think BASE_PATH should be established at the base of the project

Reply:
I moved it here because these variables are used primarily for the cogs and so when you import anything from within the cogs folder, it will initialize these variables. I am open to moving them somewhere else. I don't right off the top of my head of a standard for placing these.